src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
```

Comment:
The 'body' selector is a tag name which is generally not recommended for styling. However, exceptions are made for 'html' and 'body' tags for setting global styles, so this is acceptable.